File: src/main/java/com/java/test/rest/SampleServerRest.java
package com.java.test.rest;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/")
public class SampleServerRest extends ServerRest {

    public static void main(String[] args) throws Exception {
        new SampleServerRest().start();
    }

    @GET
    @Path("/hello")
    @Produces(MediaType.APPLICATION_JSON)
    public String hello() {
        return "Hello world";
    }
}

File: src/main/scala/com/scala/kafka.stream/rest/SampleServerRest.scala
package com.scala.test.rest

import akka.actor.ActorSystem
import spray.routing.SimpleRoutingApp

object SampleServerRest extends App with SimpleRoutingApp {

  implicit val system = ActorSystem("kafka-rest")

  startServer(interface = "localhost", port = 7070) {
    path("hello").&(get.&(complete {
      "hello world"
    }))
  }
}
